### PR TITLE
Adding endpoint to determine Code Quality Status

### PR DESCRIPTION
**Pull request by ReddytoRumble**

### CODE REVIEW

File: api-audit/src/main/java/com/capitalone/dashboard/service/CodeQualityAuditService.java [hunk 1]
```diff
@@ -2,12 +2,20 @@
...
 
 
 public interface CodeQualityAuditService {
 	
 
-    List<CodeQuality> getCodeQualityDetails(String name,String version);
+	List<Dashboard> retrieveDashboardDetails (String title);
+	
+	Cmdb retrieveBapDetails(ObjectId id);
+
```
Comment (AuditHawks) [inline api-audit/src/main/java/com/capitalone/dashboard/service/CodeQualityAuditService.java]: rename this method to align with the generic terminology that we are using - BusinessServices and Business Application.

File: api-audit/src/main/java/com/capitalone/dashboard/service/AuditService.java [hunk 2]
```diff
@@ -9,4 +9,5 @@
     List<GitRequest> getPullRequests(String repo, String branch, long beginDt, long endDt);
 
     List<Commit> getCommitsBySha (String scmRevisionNumber);
+
```
Comment (AuditHawks) [inline api-audit/src/main/java/com/capitalone/dashboard/service/AuditService.java]: @ReddytoRumble Has something changed in this file ?

File: api-audit/src/main/java/com/capitalone/dashboard/service/AuditServiceImpl.java [hunk 3]
```diff
@@ -4,9 +4,11 @@
 import com.capitalone.dashboard.model.GitRequest;
 import com.capitalone.dashboard.repository.CommitRepository;
 import com.capitalone.dashboard.repository.GitRequestRepository;
+
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
+
 import java.util.List;
```
Comment (AuditHawks) [inline api-audit/src/main/java/com/capitalone/dashboard/service/AuditServiceImpl.java]: Has something changed in this file ?

File: api-audit/src/main/java/com/capitalone/dashboard/rest/AuditController.java [hunk 4]
```diff
@@ -23,29 +30,31 @@
...
     public ResponseEntity<Iterable<PeerReviewResponse>> peerReview(@Valid PeerReviewRequest request) {
 
-        List<PeerReviewResponse> allPeerReviews = new ArrayList<PeerReviewResponse>();
+        List<PeerReviewResponse> allPeerReviews = new ArrayList<>();
 
         List<GitRequest> pullRequests = auditService.getPullRequests(request.getRepo(), request.getBranch(), request.getBeginDate(), request.getEndDate());
 
-        for(GitRequest pr : pullRequests) {
-            List commitsRelatedToPr = new ArrayList();
+        for (GitRequest pr : pullRequests) {
```
Comment (AuditHawks) [inline api-audit/src/main/java/com/capitalone/dashboard/rest/AuditController.java]: @ReddytoRumble Don't add whitespaces if you aren't changing code. It makes reviewing the code harder.

File: api/src/main/java/com/capitalone/dashboard/config/WebSecurityConfig.java [hunk 5]
```diff
@@ -137,4 +137,4 @@ protected ActiveDirectoryLdapAuthenticationProvider activeDirectoryLdapAuthentic
         return provider;
     }
 	
-}
+}
```
Comment (AuditHawks) [inline api/src/main/java/com/capitalone/dashboard/config/WebSecurityConfig.java]: @ReddytoRumble Has something changed in this file ?

**Comment by AuditHawks** [comment]
@ReddytoRumble Added some comments. Please review and fix.

**Comment by ReddytoRumble** [comment]
@AuditHawks I am going to close this and open a new PR

**Pull request closed without merging**